src/commands/scp.ts

```diff
@@ -49,6 +49,11 @@ export const scpCommand = (yargs: yargs.Argv) =>
           type: "string",
           describe: "The account on which the instance is located",
         })
+        .option("provider", {
```

Comment:
small nit (we don't need to do this now) we can create a "shared options" for ssh/scp yargs, it seems like there are enough shared args to warrant some builder function for them. Maybe we can create a ticket and give it Morph to work on?